### ISSUE TITLE
Add Filters in Meshery UI

### ISSUE THREAD

**Issue by navendu-pottekkat** [layer5io/meshery#2802]
#### Current Behavior

Meshery server now supports CRUD for WASM filters.

#### Desired Behavior

Meshery should have a Filters page on the UI which will be used work with WASM Filters. 

The UI will look similar to the Patterns UI (/configuration/patterns).

![Screenshot from 2021-05-02 10-45-08](https://user-images.githubusercontent.com/49474499/116803306-08401a80-ab34-11eb-9f86-f5f069646bf0.png)

Initially it should support CRUD for filter files similar to the current implementation of patterns and later deploy and other functionalities will be added.

One key difference with the implementation of patterns is that the edit option for a filter should only allow the user to specify a different file from a URL or Disk and **NOT** let the users edit the filter file.

#### Implementation

Implementation would be similar to the existing patterns page but including the changes mentioned above.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

**Comment by navendu-pottekkat** [comment]
Currently, it will look exactly like the pattern one. Once we have that, I guess we can move on to the specifics...

**Comment by Chadha93** [comment]
Yes, it looks exactly the same as the pattern one.  Ok, so should I push the filters.js file ? so other functionalities can be add ...

**Comment by navendu-pottekkat** [comment]
Yes, that would work @Chadha93

**Comment by Chadha93** [comment]
@navendu-pottekkat  I have added the Filter.js file. Now other functionalities can be added to it in this same issue.
